macos/docs/Controls/Link.md

```diff
@@ -0,0 +1,54 @@
+# Link
+
+## Overview
+With a Link, users can navigate to another page, window, or Help topic; display a definition; initiate a command; or choose an option. A Link indicates that it can be clicked, typically by being displayed using the visited or unvisited link system colors. Traditionally, Links are underlined as well, but that approach is often unnecessary and falling out of favor to reduce visual clutter.
+
+A Link is the lightest weight clickable control, and is often used to reduce the visual complexity of a design.
+
+
+
+## Best Practices
+### Do
+- Use to navigate to another page, window, or help topic; display a definition; initiate a command; or choose an option.
+- Make Links discoverable by visual inspection alone. Users shouldn't have to interact with your program to find links.
+- Use Links that give specific descriptive information about the result of clicking on the link, using as much text as necessary. Users should be able to - accurately predict the result of a link from its link text and optional Tooltip.
+- Use text that suggests clicking, such as a command starting with an imperative verb like "Show", "Print", "Copy", or "Delete".
+
+### Don't
```

Comment:
Probably also want a "Don't" for use link for navigation internal to your app? That conflicts with the valid help case, but there's gotta be some wording to explain that these should be used for navigating to external content and buttons should be used instead in most cases